podium-gateway/src/main/webapp/app/shared/status-update/request-update-status-dialog.component.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2017. The Hyve and respective contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * See the file LICENSE in the root of this repository.
+ *
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { JhiLanguageService } from 'ng-jhipster';
+import { RequestService } from '../request';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { PodiumEventMessage } from '../event/podium-event-message';
+import { RequestUpdateDialogComponent } from './request-update-dialog.component';
+import { RequestUpdateAction } from './request-update-action';
+
+@Component({
+    templateUrl: './request-update-dialog.component.html',
+    styleUrls: ['request-update-dialog.scss']
+})
+
+export class RequestUpdateStatusDialogComponent extends RequestUpdateDialogComponent implements OnInit {
+    statusUpdateAction: RequestUpdateAction;
+    headerStyle: string;
+    buttonStyle: string;
+    status: string;
+    public message: PodiumEventMessage = new PodiumEventMessage();
+
+    constructor(protected jhiLanguageService: JhiLanguageService,
+                protected requestService: RequestService,
+                protected activeModal: NgbActiveModal) {
+        super(jhiLanguageService, requestService, activeModal);
+    }
+
+    ngOnInit() {
+        this.applyStyles();
+    }
+
+    applyStyles() {
```

Comment:
Since this is duplicated close with the other dialog that extends the same super class, you could move this functionality to the parent class. If that is the only benefit of having that level of inheritance, please consider removing it